api/model.py

```diff
@@ -39,6 +39,7 @@ def get(self):
 audio_parser = api.parser()
 audio_parser.add_argument('audio', type=FileStorage, location='files', required=True,
                           help="signed 16-bit PCM WAV audio file")
+audio_parser.add_argument('start_time', type=float, default=0, help='The time (in seconds) in the audio file to run prediction at.')
```

Comment:
Maybe the help text could be something like "The number of seconds into the audio file the prediction should start at"

Reply:
That's fine by me, I'll update it